### DISCUSSION TITLE
2006/09/11/labview-%E7%9A%84%E8%B0%83%E8%AF%95%E7%8E%AF%E5%A2%83

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#114] [Announcements]
# 2006/09/11/labview-%E7%9A%84%E8%B0%83%E8%AF%95%E7%8E%AF%E5%A2%83

1\. LabVIEW 的全局选项

https://blog.qizhen.xyz/2006/09/11/labview-%E7%9A%84%E8%B0%83%E8%AF%95%E7%8E%AF%E5%A2%83

**Comment by ruanqizhen** [comment]
之前博客下的留言：

橙色
2006年09月12日 14:35 编辑
我来了，前几天SP出问题，一直没上来。
你在我的朋友列表，所以我一上来就到处看你的留言，想看看我有没到过你这里的痕迹，
可是找了半天，也没有，不好意思，现在才来看你。。
你空间里面的东西好专业，你是做软件的吗？
